dataService.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-undef */
+var product = require('./src/data/products.json');
+var category = require('./src/data/categories.json');
+
+let p = product.products;
+let c = category.categories;
+
+global.productMap = new Map();
```

Comment:
Do we actually need `global` prefix here?

Reply:
I misread the instructions and thought the task required variables to be accessible across all files in Node js. Edited now, thanks!